js/battledata.js

```diff
@@ -396,21 +396,12 @@ var Tools = {
 					'" target="_blank" onclick="' + onclick + '">' + uri + '</a>';
 			});
 			// google [blah]
-			// google[blah]
```

Comment:
I removed this line note because it is now the only entry that indicates the space is optional with a line note, so I felt it was not only unnecessary but also inconsistent with the other entries.